tests/test_modeling_tf_rembert.py

```diff
@@ -90,8 +90,7 @@ def __init__(
         self.num_choices = 4
         self.scope = None
 
-        # RemBERT also returns the upprojected word embeddings as an hidden layers
-        self.expected_num_hidden_layers = self.num_hidden_layers + 2
+        self.expected_num_hidden_layers = self.num_hidden_layers + 1
```

Comment:
I think this line can now be removed as it's the default.

Reply:
ok